vpn-router-web/src/main/java/vpnrouter/web/ClientsList.java

```diff
@@ -0,0 +1,182 @@
+package vpnrouter.web;
+
+import com.vaadin.componentfactory.ToggleButton;
+import com.vaadin.flow.component.AttachEvent;
+import com.vaadin.flow.component.Component;
+import com.vaadin.flow.component.Focusable;
+import com.vaadin.flow.component.applayout.AppLayout;
+import com.vaadin.flow.component.button.Button;
+import com.vaadin.flow.component.button.ButtonVariant;
+import com.vaadin.flow.component.confirmdialog.ConfirmDialog;
+import com.vaadin.flow.component.dependency.CssImport;
+import com.vaadin.flow.component.grid.Grid;
+import com.vaadin.flow.component.grid.editor.Editor;
+import com.vaadin.flow.component.html.H3;
+import com.vaadin.flow.component.orderedlayout.VerticalLayout;
+import com.vaadin.flow.component.textfield.TextField;
+import com.vaadin.flow.data.binder.Binder;
+import com.vaadin.flow.data.renderer.ComponentRenderer;
+import com.vaadin.flow.router.Route;
+import com.vaadin.flow.spring.annotation.UIScope;
+import lombok.extern.slf4j.Slf4j;
+import vpnrouter.api.client.ClientService;
+import vpnrouter.api.client.ClientUpdate;
+import vpnrouter.api.client.ClientView;
+
+import java.util.List;
+
+@CssImport("./styles/styles.css")
+@UIScope
+@Route("clients")
+@Slf4j
+public class ClientsList extends AppLayout {
```

Comment:
Well, in Java List is a data structure. In front-end List is a UI component (as far as I know). This class doesn't represent a List. It should be named ClientsPage, or ClientsView, or smth like that.